README.md

```diff
@@ -65,16 +65,26 @@ docker pull xaionaro2/edk2-builder:RefindPlusUDK
 # hacky fix for duplication error of lodepng_malloc and lodepng_free
 sed -e 's/void[*] lodepng_malloc/void* _dup_lodepng_malloc/' \
     -e 's/void lodepng_free/void _dup_lodepng_free/' \
+	-e 's/void[*] lodepng_refit_malloc/void* _dup_lodepng_refit_malloc/' \
+    -e 's/void lodepng_refit_free/void _dup_lodepng_refit_free/' \
     -i-orig RefindPlusPkg/libeg/lodepng_xtra.c
 
-# building
+# hacky fix to remove "STATIC_ASSERT undefined" error
```

Comment:
Can't verify either, but this STATIC_ASSERT issue seems to be down to outdated RefindPlusUDK

Reply:
This hack is not needed indeed.